src/index.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from './App';
+import { BrowserRouter as Router } from 'react-router-dom';
+import Home from './pages/Home';
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    {/* <App /> */}
+    <Router basename={process.env.PUBLIC_URL}>
+      <Home />
+    </Router>
```

Comment:
- [ ] Router is not required here as you already have the Routes in your `Home` component. Kindly remove the Router from here.

Reply:
- Router is just an alias for BroserRouter,
- basename is required so all routes have the same base URL (in this case the homepage property from package.json.